tools/snp.sh

```diff
@@ -1113,6 +1250,7 @@ main() {
       ;;
 
     setup-host)
+      verify_if_host_is_snp_capable
```

Comment:
Wouldn't it make sense to have a single function for this and pass a parameter to determine guest/host?

Reply:
As per this, I created a single function "verify_platform_snp_bit_status" with guest/host flag to handle snp bit check